module/Application/view/application/index/index.phtml

```diff
@@ -4,7 +4,7 @@
     <p>
         Congratulations! You have successfully installed the
         <a href="https://github.com/zendframework/ZendSkeletonApplication" target="_blank">ZF Skeleton Application</a>.
-        You are currently running Zend Framework version <?= \Application\Module::VERSION ?>.
+        You are currently running Zend Framework 3 with zend-mvc version <?= \PackageVersions\Versions::getShortVersion('zendframework/zend-mvc') ?>.
```

Comment:
`\PackageVersions\Versions::getVersion('zendframework/zend-mvc')` is ok to get full version with hash included